src/drive/styles/actionmenu.styl

```diff
@@ -48,7 +48,7 @@
     background embedurl("../assets/icons/icon-share-grey.svg") 1rem center no-repeat
     position   relative
     
-    > div
+    [class*='menu-recipients']
```

Comment:
I think this sort of selector is dangerous. It's too easy to add a class that contains `menu-recipients` somewhere and accidentally have these rules applied.
Since we use it in 2 different places, I think it's worth having a dedicated class name.